ansible/files/logstash-fsf-es.conf

```diff
@@ -0,0 +1,27 @@
+input {
+  file {
+    codec => "json"
+    path => "/data/fsf/scan.log"
+    add_field => { "[@metadata][stage]" => "fsf" }
+  }
+}
+
+filter {
+  if "_jsonparsefailure" in [tags] {
+    drop { }
+  }
+  if [@metadata][stage] == "fsf" {
+    mutate { remove_field => ["path"] }
+  } 
+}
+
+output {
+  if [@metadata][stage] == "fsf" {
+    #stdout { codec => rubydebug }
+    elasticsearch {
```

Comment:
Elasticsearch doesn't like the output of fsfclient when used with the test sample. I get the following from Logstash when it tries to push it:
```
[2017-02-05T22:06:58,238][WARN ][logstash.outputs.elasticsearch] Failed action. {:status=>400, :action=>["index", {:_id=>nil, :_index=>"fsf-2017.02.05", :_type=>"fsf", :_routing=>nil}, 2017-02-05T22:06:58.076Z simplerockbuild.simplerock.lan %{message}], :response=>{"index"=>{"_index"=>"fsf-2017.02.05", "_type"=>"fsf", "_id"=>"AVoQUNhPfY0xdLD9uEUK", "status"=>400, "error"=>{"type"=>"illegal_argument_exception", "reason"=>"mapper [Object.EXTRACT_ZIP.Object_16.EXTRACT_ZIP.Object_10.META_JAVA_CLASS.constants_pool] of different type, current_type [text], merged_type [long]"}}}}
```